### DISCUSSION TITLE
推荐开发"一键同步更新mod"

### DISCUSSION THREAD

**Discussion by xiaowu114** [Meloong-Git/PCL#3301] [讨论]
大概mod栏可以放一个“更新此mc版本mod”
1是希望支持版本mod同步：每次新mc版本下载所有mod又要重新下一遍，还容易下错文件夹，版本错了也不方便查
2是希望同步最新mod：mod更新又不方便一个一个去下载
如果有发现无次版本的mod，做个窗口提示有哪些未成功实现的

**Comment by allMagicNB** [comment]
#102

**Comment by Enigma-Soul** [comment]
因为MOD分兼容性，有可能会出现1版本可以共存，但2版存不了，**当然，机率很小，所以如果做了debug也是可以的**
